persistence/src/main/java/org/mskcc/cmo/metadb/persistence/CmoRequestRepository.java

```diff
@@ -12,6 +14,16 @@
  */
 @Repository
 public interface CmoRequestRepository extends Neo4jRepository<CmoRequestEntity, Long> {
-    @Query("CREATE (n:cmo_metadb_request {requestId: $requestId})")
-    public void saveNewCmoRequest(@Param("requestId") String requestId);
+    @Query("MATCH (r:cmo_metadb_request {requestId:$reqId}) RETURN r;")
```

Comment:
would you mind adjusting the white space to match the format from the method we had before? 

so `requestId:$reqId` -->`requestId: $reqId` for example

and apply wherever necessary below as well to keep things a bit easier to read/clean